inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -72,7 +72,7 @@ ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolat
         attrsV4.pads_begin = attrsV0.pads_begin;
         attrsV4.pads_end = attrsV0.pads_end;
         attrsV4.antialias = attrsV0.antialias;
-        attrsV4.coordinate_transformation_mode = ngraph::opset4::Interpolate::CoordinateTransformMode::half_pixel;
+        attrsV4.coordinate_transformation_mode = ngraph::opset4::Interpolate::CoordinateTransformMode::asymmetric;
```

Comment:
maybe we can set attrsV4.nearest_mode = ngraph::opset4::Interpolate::NearestMode::simple as well, now it's attrsV4.nearest_mode = ngraph::opset4::Interpolate::NearestMode::round_prefer_floor.

Reply:
nearest_mode == simple by default now